website/pages/en/substreams.mdx

```diff
@@ -4,43 +4,36 @@ title: Substreams
 
 ![Substreams Logo](/img/substreams-logo.png)
 
-Substreams is a powerful blockchain indexing technology developed for The Graph Network. It enables developers to write Rust modules, compose data streams alongside the community, and provide extremely high-performance indexing due to parallelization in a streaming-first approach.
+Substreams is a powerful blockchain indexing technology developed to improve The Graph Network:
+- Substreams allows you to speed up the indexing times of your subgraph.
+- Substreams allows you to index non-EVM data (Solana, Injective, Starknet, Vara...)
 
-With Substreams, developers can quickly extract data from different blockchains (Ethereum, BNB, Solana, ect.) and send it to various locations of their choice, such as a Postgres database, a Mongo database, or a Subgraph. Additionally, Substreams packages enable developers to specify which data they want to extract from the blockchain.
+This is possible because your subgraph can use a Substreams package as a data source, thus creating a Substreams-powered Subgraph (SpS).
```

Comment:
```suggestion
This is possible because your subgraph can use a Substreams package as a data source, thus creating a Substreams-powered subgraph.
```
According to "The Graph Style Guide", we shouldn't capitalize "subgraph", even in the phrase "Substreams-powered subgraphs".

(@bermchain, should we make that document public, and maybe link to it from the docs' CONTRIBUTING guidelines?)

Reply:
Oh and I propose removing the acronym because it's not used anywhere else on this page, but maybe it's used somewhere else and I'm not aware?

Reply:
Agreed @benface with lowercase _subgraph_ and not using the acronym unless it's recurring.

I like the idea of publicizing the style guide in the Docs! Let's do it -- I'll select sections that make sense to share.

Also, re: "Substreams is" vs. "Substreams are" -- I believe it's the former because Substreams is a collective noun for the technology (i.e., powered by Substreams tech, not powered by _a_ Substream).